src/Microsoft.VisualStudio.Editor.Razor/VisualStudioDocumentTrackerProvider.cs

```diff
@@ -1,12 +1,13 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using Microsoft.CodeAnalysis.Host;
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Editor;
 
 namespace Microsoft.VisualStudio.Editor.Razor
 {
-    public abstract class VisualStudioDocumentTrackerFactory
+    public abstract class VisualStudioDocumentTrackerProvider : ILanguageService
```

Comment:
Can we just delete the work provider from the world? I'm not happy to see it popping up all over. It doesn't mean anything

Reply:
lol, i'll rename it `VisualStudioHelperDocumentTrackerHelperHelper`. If you don't like provider please provide a good alternative.

Reply:
nm, you did with Accessor. Will rename